ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/activity/PaywallActivity.kt

```diff
@@ -123,6 +180,26 @@ internal class PaywallActivity : ComponentActivity(), PaywallListener {
         setResult(RESULT_OK, createResultIntent(PaywallResult.Error(error)))
     }
 
+    private fun configureSdkWithSavedData(savedInstanceState: Bundle) {
+        val sdkConfigArgs = getSdkConfigArgs(savedInstanceState)
+        if (sdkConfigArgs == null) {
+            Logger.e("TEST: Missing SDK configuration arguments")
```

Comment:
Good log, but maybe without the "TEST"? 😛

Reply:
😅 Yeah my bad, I didn't clean it up :P